core/src/main/java/org/commonjava/maven/ext/core/impl/DependencyManipulator.java

```diff
@@ -793,6 +804,8 @@ else if ( strict && oldVersion.contains( "$" ) &&
                 final boolean isModuleWildcard = currentKey.endsWith( "@*" );
                 logger.debug( "Is wildcard? {} and in module wildcard mode? {} ", isModuleWildcard, aWildcardMode );
 
+                String artifactGA = null;
```

Comment:
Given its usage shouldn't this be immediately converted to a ProjectRef on construction? (to avoid repeated conversions)

Reply:
For sure, although fixing this existing code repetition would have muddied the diff a bit.